src/math/FGFunction.h

```diff
@@ -689,10 +688,24 @@ refers to one or more instances of a property, value, or table.
      </switch>
      @endcode
 - @b random Takes no arguments and returns a Gaussian distributed random number
-    @code <random/> @endcode
+    @code
+    <random/> 
+    Default configuration with mean: mean="0.0" and standard deviation: stddev="1.0"
+    Optional parameters:
+    <random seed="1234"/>
+    <random seed="time_now"/>
+    <random seed="time_now" mean="0.0" stddev="1.0"/>
+    @endcode
 - @b urandom, Takes no arguments and returns a uniformly distributed random
               number between -1 and +1
-    @code<urandom/>@endcode
+    @code
+    <urandom/>
+    Default configuration with lower value: lower="-1.0" and upper value: upper="1.0"
```

Comment:
Do not include comments inside `@code` / `@endcode` pairs
```suggestion
    @endcode
    Default configuration with lower value: lower="-1.0" and upper value: upper="1.0"
```